### ISSUE TITLE
CMD-S to save on Mac

### ISSUE THREAD

**Issue by Phantom75** [jsreport/jsreport#559]
Feature Request: It would be great if Ctrl-S saves the currently selected object (template, data object, css, etc.), just as if the "Save" button were being hit.

**Comment by pofider** [comment]
This is already supported for very long time.

Have you tried it?

**Comment by Phantom75** [comment]
The issue is that I am using a mac. We are used to hit CMD-S to save which pops up the dialog to save the HTML-Page. Ctrl-S works indeed but on Mac, the cmd-S is burned into users habbits. Let me rephrase my request:
It would be awsome, if save on Mac would be possible by the key command CMD-S :)

**Comment by bjrmatos** [comment]
i second this, i'm also a Mac user and i'm used to do CMD + S in other editors, it is the nature shortcut on that OS, i guess there is no harm in supporting this shortcut

**Comment by pofider** [comment]
Ok, we will take a look at it.

**Comment by Phantom75** [comment]
Man, that's fast! Kudos and hope, I can soon reward this with a license.
